[CES-3323] Set NGP Proto screens to the correct touch orientation

bryanfoley opened the pull request:
## Description
With this PR, if a printer has the BOM number 109*, then it is an NGP printer and uses a large Colorado like screen, except it is not an inverted screen like the production printers.

So we use a different dts for machines with BOM's 109* which is very similar to the ulticontroller4.0-lvds-1024x600.dts, just the inversion of the x and y is removed.

## How has this been tested
Installed this um-kernel package on NGP proto, after install teh touch response of the screen is correct.

## Ready for Review Checklist
To help with deciding if this PR is RFR, use this checklist.

The author confirms that:
- [x] the author has **self-reviewed** this work and is highly confident about the quality
- [x] this work satisfies all **acceptance criteria** that are stated in the linked ticket
- [x] this work has been **tested** on all product families and the process and results are documented in the above section
- [x] The **description** above is concise yet complete
- [ ] the reviewer has been offered a **walkthrough** (if needed)
- [x] the code is **cleaned** from any rubbish (e.g. meaningless comments, log-spamming, etc...)
- [x] remaining `#TODO` **comments** mention a Jira ticket number
- [x] all **CI** checks are passing
- [x] all **commits** are (re)structured to be meaningful and clearly arranged, and all are prepended with the ticket number for traceability

Commit:
[CES-3322] Set NGP Proto screens to the correct touch orientation

With this PR, if a printer has the BOM number 109*, then it is an NGP
printer and uses a large Colorado like screen, except it is not an
inverted screen like the production printers.

So we use a different dts for machines with BOM's 109* which is very
similar to the ulticontroller4.0-lvds-1024x600.dts, just the inversion
of the x and y is removed.

robinmdh reviewed (approved):
LGTM, however please note that I can't actually test this, as I don't have an NG printer to test on.

Commit:
[CES-3322] Decrease the contents of ulticontroller4.1-lvds-1024x600.dts

With this commit we reduce the contents of this file so that it only
removes the touch screen inversion.
